src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,20 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
```

Comment:
[Add a private default constructor to HibernateUtil class in order to prevent creating HibernateUtil objects.](https://mate-academy.github.io/jv-program-common-mistakes/hibernate/relations/relations_checklist)